projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -381,6 +381,7 @@ if (environment.cpq) {
         enableConsecutiveCharactersPasswordRequirement: true,
         enablePasswordsCannotMatchInPasswordUpdateForm: true,
         allPageMetaResolversEnabledInCsr: true,
+        cdsCasEnabled: false,
```

Comment:
Ideally, all new toggles should be disabled by default for customers, but **enabled in our example storefrontapp** (for more, see our [Coding Guidelines](https://wiki.one.int.sap/wiki/display/spar/Coding+Guidelines#CodingGuidelines-AddaFeatureToggleinthecodesourcecode)).

But perhaps you might have some strong arguments to not enable this particular toggle in our storefrontapp. Then please help me understand those arguments.

Reply:
I have set it to true. As you said, it makes more sense to have it enabled by default. We want new customers to use the new way only.